### PR TITLE
Add use case, reduce NFR in DG

### PR DESCRIPTION
**Pull request by swtan346**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -605,31 +621,20 @@ specified otherwise)
...
-19.  The product should process a user input command within 2 second.
+1.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) 
+    should be able to accomplish most of the tasks faster using commands than using the mouse.
+1.  The codebase should be well-structured and well-documented to facilitate future maintenance and enhancements.
+1.  The application should only support a single user.
+1.  The data should be stored locally and should be in a human editable text file.
+1.  The software should work without requiring an installer.
+1.  The software should not depend on any specific remote server.
+1.  The product should be available as a single JAR file of size 100MB or below.
+1.  The product should process a user input command within 2 second.
```
Comment (iamtr) [inline docs/DeveloperGuide.md:640]: numbering here is incorrect

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -523,13 +523,13 @@ specified otherwise)
 * 2a. The given index is invalid.
   * 2a2. AddressBook shows an error message.
 
-    Use case resumes at step 2.
+    Use case ends.
 
 **Use case: `US04 - Edit a patient records`**
```
Comment (iamtr) [inline docs/DeveloperGuide.md]: Should be UC instead of US

**Review by iamtr — COMMENTED** [review]
A few changes may be required

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (swtan346) [reply docs/DeveloperGuide.md:640]: markdown coding standard

image

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (swtan346) [reply docs/DeveloperGuide.md]: ok

**Review by erv-teo — APPROVED** [review]
LGTM

**Pull request merged**